uh_internal/apps/dailyduties/pyexchange.py

```diff
@@ -0,0 +1,39 @@
+import os
+from os.path import join, dirname
+from ...settings.base import get_env_variable
+
+# https://github.com/fedorareis/pyexchange This is a combination of a few branches and some custom code
+from pyexchange import Exchange2010Service, ExchangeNTLMAuthConnection, ExchangeBasicAuthConnection
+
+URL = get_env_variable('RESNET_INTERNAL_OUTLOOK_URL')
+USERNAME = get_env_variable('RESNET_INTERNAL_EMAIL_OUT_USERNAME')  # ResNet username
+PASSWORD = get_env_variable('RESNET_INTERNAL_EMAIL_OUT_PASSWORD')  # ResNet Password
```

Comment:
These should all be set in settings and imported. Also, make them more descriptive, like ``OUTLOOK_URL``, ``EMAIL_OUT_USERNAME``, etc.